schedule/yast/maintenance/create_hdd_transactional_server_dev.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: create_hdd_transactional_server
+description: >
+    Installation of a Transactional Server which uses a read-only
+    root filesystem to provide atomic, automatic updates of a
+    system without interfering with the running system.
+vars:
+    DESKTOP: textmode
+    PATTERNS: base,minimal
+    HDDSIZEGB: 20
+    DUD_ADDONS: sdk
+    YUI_REST_API: 1
+schedule:
+    - installation/bootloader_start
+    - installation/setup_libyui
+    - installation/product_selection/install_SLES
+    - installation/licensing/accept_license
+    - installation/registration/register_via_scc
+    - installation/module_registration/register_module_desktop
```

Comment:
we are selecting HA by mistake using this test module because probably we didn't prepare for having a `,` at the end:
`SCC_ADDONS: serverapp,base,tsm,`
it is out of scope to fix this, you can set this module and file a ticket for this, or if you see easy fix, probably it is just and undef value in a list that we process and we should just remove it :)

Reply:
I tried to delete unexpected element add code '@scc_addons = grep { $_ ne '' } @scc_addons;' It still reported error: Use of uninitialized value $module_name in regexp compilation at /var/lib/openqa/pool/4/os-autoinst-distri-opensuse/lib/Installation/ModuleRegistration/ModuleRegistrationPage.pm line 71.
	Installation::ModuleRegistration::ModuleRegistrationPage::register_module(Installation::ModuleRegistration::ModuleRegistrationPage=HASH(0x5618294f9688), "tsm") called at /var/lib/openqa/pool/4/os-autoinst-distri-opensuse/lib/Installation/ModuleRegistration/ModuleRegistrationPage.pm line 77
This is because it cannot know 'tsm'.  At  'installation/module_registration/register_module_transactional' it inputs module name as '$testapi::distri->get_module_registration()->register_module('transactional');' . But at our setting, it uses 'tsm'. It cannot know 'tsm'.  I will put back to use 'installation/module_registration/register_module_transactional'. Will open new ticket to fix 'tsm' problem. Failed case: https://openqa.suse.de/tests/9720939#

Reply:
tsm is not mapped, this should be renamed:
`$self->{rct_item_transactional}`

Reply:
Updated the code, it can register module 'tsm'. But the workflow in 'register_extensions_and_modules' is different with 'register_module_transactional'. It cannot be used at here. Refer: https://openqa.suse.de/tests/9725085#step/register_extensions_and_modules/2

Reply:
I will keep change about 'tsm' problem. But at this yaml file, I'd like to use 'installation/module_registration/register_module_transactional' instead of 'installation/module_registration/register_extensions_and_modules'